avogadro/qtplugins/atompairbonds/atompairbonds.h

```diff
@@ -0,0 +1,60 @@
+/******************************************************************************
+  This source file is part of the Avogadro project.
+  This source code is released under the 3-Clause BSD License, (see "LICENSE").
+******************************************************************************/
+
+#ifndef AVOGADRO_QTPLUGINS_ATOMPAIRBONDS_H
+#define AVOGADRO_QTPLUGINS_ATOMPAIRBONDS_H
+
+#include <avogadro/core/vector.h>
+#include <avogadro/qtgui/sceneplugin.h>
+
+namespace Avogadro {
+namespace QtPlugins {
+
+/**
+ * @brief Predict atom-pair interactions, like hydrogen bonds.
+ * @author Aritz Erkiaga
+ */
+class AtomPairBonds : public QtGui::ScenePlugin
+{
+  Q_OBJECT
+
+public:
+  explicit AtomPairBonds(QObject* parent = nullptr);
+  ~AtomPairBonds() override;
+
+  void process(const QtGui::Molecule& molecule,
+               Rendering::GroupNode& node) override;
+
+  QString name() const override { return tr("Atom-Pair Bonds"); }
```

Comment:
I'm tempted to call it "Hydrogen Bonds" or "Non-Covalent" maybe.

Reply:
"Non-covalent" seems good. The idea is that the same plugin should handle halogen and chalcogen bonds too. All of these have in common the fact they involve an atom and an electron pair from another atom, unlike salt bridges etc., but to my knowledge no one calls them "Atom-pair bonds", so I wholeheartedly agree.

Should I change all references to atom-pair interactions in the code, or just the name?